single-user-d4science/Dockerfile

```diff
@@ -67,5 +69,5 @@ RUN pip install --no-cache-dir \
 
 RUN julia -e "using Pkg; Pkg.pin(\"IJulia\"); pkg\"add https://github.com/gher-ulg/DIVAndNN.jl\"; pkg\"add JSON PyCall PyPlot DIVAnd Glob DataStructures NCDatasets\""
 
-RUN jupyter labextension install jupyterlab-topbar-extension jupyterlab-system-monitor
-RUN jupyter labextension disable @jupyterlab/filebrowser-extension:share-file
+#RUN jupyter labextension install jupyterlab-topbar-extension jupyterlab-system-monitor
+#RUN jupyter labextension disable @jupyterlab/filebrowser-extension:share-file
```

Comment:
this one caused confusion in the past. I would keep it disabled unless needed.

Reply:
but  now it's there...i made a mistake when pushing initially